packages/common/src/ui/components/navigation/NavLink/NavLink.tsx

```diff
@@ -48,25 +49,82 @@ const StyledListItem = styled<
 export interface NavLinkProps {
   end?: boolean; // denotes lowest level menu item, using terminology from useMatch
   icon?: JSX.Element;
+  expandOnHover?: boolean;
   text?: string;
   to: string;
 }
 
 export const NavLink: FC<NavLinkProps> = props => {
-  const { end, icon = <span style={{ width: 2 }} />, text, to } = props;
+  const {
+    end,
+    icon = <span style={{ width: 2 }} />,
+    expandOnHover,
+    text,
+    to,
+  } = props;
   const drawer = useDrawer();
   const selected = useSelectedNavMenuItem(to, !!end);
 
   const CustomLink = React.useMemo(
     () =>
-      React.forwardRef<HTMLAnchorElement>((linkProps, ref) => (
-        <Link {...linkProps} ref={ref} to={to} role="link" aria-label={text} />
-      )),
+      React.forwardRef<HTMLAnchorElement>((linkProps, ref) =>
+        expandOnHover && !end ? (
```

Comment:
Will there be top level nav items here with no sub menu? Maybe the dashboard? It would be even weirder if everything looked the same, and even the placement couldn't inform you if something was clickable